src/Migrations/Utils/Functions.php

```diff
@@ -14,13 +15,17 @@ class Functions
      *
      * @param string $block_name - The name of the block to be searched.
      * @param array $post_types - The list of post types to look for.
+     * @param array $post_status - The list of post status to look for.
      * @return mixed - The posts using a type of block or null if no posts are found.
      */
-    public static function get_posts_using_specific_block(string $block_name, array $post_types): mixed
+    //phpcs:ignore Generic.Files.LineLength.MaxExceeded
+    public static function get_posts_using_specific_block(string $block_name, array $post_types, array $post_status): mixed
```

Comment:
I think we have to make the `$post_status` parameter optional.
Otherwise, there will be issues with other parts of the code using this function (check the `src/Migrations/M031MigrateMediaBlockToAudioVideoBlock.php` file) and the migrator will crash.

Reply:
@sagarsdeshmukh you can check my implementation for this here: https://github.com/greenpeace/planet4-master-theme/blob/cf8b234ce4490117345c6506016ccec6afd69ee6/src/Migrations/Utils/Functions.php#L23